src/main/java/org/tl/HobbiesTest.java

```diff
@@ -0,0 +1,82 @@
+package org.tl;
+
+import lombok.RequiredArgsConstructor;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@RequiredArgsConstructor
+public class HobbiesTest {
+
+    private Hobbies hobbies;
+    private String testFilePath;
+
+
+    @BeforeEach
+    void setUp() {
+        hobbies = new Hobbies();
+        testFilePath = "src/test/resources/hobbies.txt";
+    }
+
+    @Test
+    void testCreateListFromFile() {
+        List<String> actualResult = hobbies.createListFromFile(testFilePath);
+        assertEquals(200, actualResult.size());
+
+    }
+
+    @Test
+    void testCreateDictionary() {
+        Map<String, List<String>> actualResult = hobbies.createDictionary(testFilePath);
+        Map<String, List<String>> expectedResult = new HashMap<>();
+        expectedResult.put("Olivia", Arrays.asList("Traveling"));
+        expectedResult.put("Aiden", Arrays.asList("Skiing"));
+        expectedResult.put("Lucas", Arrays.asList("Cycling"));
+        expectedResult.put("Isabella", Arrays.asList("Hiking"));
+        expectedResult.put("Jackson", Arrays.asList("Skiing"));
+        expectedResult.put("Ava", Arrays.asList("Drawing"));
+        expectedResult.put("Liam", Arrays.asList("Cycling"));
+        expectedResult.put("Noah", Arrays.asList("Traveling"));
+        expectedResult.put("Emma", Arrays.asList("Playing Guitar"));
+        expectedResult.put("Sophia", Arrays.asList("Cooking"));
```

Comment:
Your tests pass but your tests are also wrong :(
Check the file and you will see, that there are people with multiple hobbies